doc/latex/Development.tex

```diff
@@ -36,52 +36,55 @@ \section{General Information} \label{sec:general-information}
 that changes to the code do not cause unintended behaviour or
 unexplained differences in the model results, so the development
 version is normally safe to use. However, it is recommended to use the
-stable version of AtChem2 for ``production runs'' and publications
-since it can be more easily referenced -- see the discussion about
-traceability and reproducibility of computational models in
-\citet{sommariva_2020}. For information on how to reference the model,
-go to Sect.~\ref{sec:how-to-cite}.
+stable version for ``production runs'' and publications since it can
+be more easily referenced -- see the discussion about traceability and
+reproducibility of computational models in \citet{sommariva_2020}.
 
 Feedback, bug reports, comments and suggestions are welcome: the list
 of open issues and known bugs can be found on the related
 \href{https://github.com/AtChem/AtChem2/issues}{github page}. The
 preferred way to contribute to the development of AtChem2 is to use
-\textbf{git} with a \href{https://github.com}{github.com} account. A
-working \href{https://swcarpentry.github.io/git-novice/}{knowledge} of
-git is a \emph{requirement}. Instructions on how to set up git and
-contribute to the model development can be found on the
-\href{https://github.com/AtChem/AtChem2/wiki/How-to-contribute}{wiki}; the
-coding guidelines for Fortran are available in Sect.\ref{sec:style-guide}.
+\textbf{git}: instructions on how to set up git and submit
+contributions can be found on the
+\href{https://github.com/AtChem/AtChem2/wiki/How-to-contribute}{wiki}.
+The coding guidelines for Fortran are detailed in Sect.\ref{sec:style-guide}.
 
 % -------------------------------------------------------------------- %
 \section{Test Suite} \label{sec:test-suite}
 
 AtChem2 uses \href{https://travis-ci.org/}{Travis CI} for
-\textbf{Continuous Integration} testing. This programming approach
+\textbf{continuous integration} testing. This programming approach
 ensures that changes to the code do not modify the behaviour and the
 results of the software in an unintended fashion. The Test Suite is in
 the \texttt{travis/} directory and consists of a series of tests and
 short model runs that check the model functionality and calculations
-against known outputs. There are four types of tests, which can be
-executed from the \maindir\ using the \verb|make| command (the
+against known outputs. The \href{https://codecov.io/}{Codecov} service
+is used to verify that the Test Suite covers a significant fraction of
+the codebase ($>$90\%) and a wide range of common configurations.
+
+There are five types of tests, which can be executed from the
+\maindir\ using the \verb|make| command (note that the
 \hyperref[subsec:optional-dependencies]{optional dependecies} need to
 be installed):
 
 \begin{itemize}
+\item \textbf{Build} test: checks that the model compiles and runs
```

Comment:
L66 talks about these tests being executed by `make` - I don't think the build test can be?

Reply:
Yes you are right. i thought `make alltests` also did that but it is not the case. It is only Travis that runs this additional test.